D1673/P1673.md

```diff
@@ -3469,6 +3506,40 @@ template<class ElementType,
 constexpr auto conjugate_transposed(
   mdspan<ElementType, Extents, Layout, Accessor> a);
 
+// [linalg.concepts], exposition-only concepts
+
+template<class T>
+struct @_is-mdspan_@; // exposition only
```

Comment:
Should it say "concept" instead of `struct`?

Reply:
Thanks for taking a look at this!  Unlike the other concepts here, `is-mdspan` is a struct (a trait following the usual pattern of inheriting from `true_type` or `false_type`).  On the other hand, this means that it would make sense for me to title this section "exposition-only concepts and traits."

Reply:
@dalg24 I changed the title; thanks!